### ISSUE TITLE
Update doesn't work when passing an array as a field value

### ISSUE THREAD

**Issue by dantrevino** [dannyconnell/localbase#1]
Love this library, but cant figure out what I'm doing wrong here.  Thanks in advance.

My doc looks like this:
```json
{
  "id":"bdfa1a2b-7950-4c97-935e-a4f37d99345b",
  "title":"",
  "text":"",
  "color":"white",
  "notebook":"",
  "modified":"2020-09-20T06:44:58.587Z",
  "created":"2020-09-20T06:44:58.587Z",
  "fave":false,
  "trash":false,
  "tags":[],
  "shares":[],
  "attachments":[]
  }
}
```

I add it successfully, no issues
![Screenshot_20200920_001843](https://user-images.githubusercontent.com/7738/93705974-e3cc8c00-fad6-11ea-9908-5145cfd89856.png)

But when I go to update it, I get the following error:
```red
Uncaught ReferenceError: update is not defined
    at updateObject (updateObject.js?c9f3:6)
    at eval (update.js?0aaf:19)
    at IDBRequest.req.onsuccess (localforage.js?a002:979)
```
Here is the update code:
```javascript
    UPDATE_NOTE(state, payload) {
      console.log("update payload: ", payload);
      db.collection("NoteRiot")
        .doc({ id: payload.id })
        .update(payload.updates)
        .then((response) => {
          console.log("note update res: ", response);
          let idx = state.notes.findIndex((note) => note.id === payload.id);
          let update = Object.assign(state.notes[idx], payload.updates);
          state.notes.splice(idx, 1, update);
        })
        .catch((error) => {
          console.log("error updating local db: ", error);
        });
    },
```
and update payload content:
```
update payload:  
{id: "bdfa1a2b-7950-4c97-935e-a4f37d99345b", updates: {…}}
id: "bdfa1a2b-7950-4c97-935e-a4f37d99345b"
updates:
attachments: Array(0)
color: "white"
created: 1600586238190
fave: false
id: "bdfa1a2b-7950-4c97-935e-a4f37d99345b"
modified: 1600586247499
notebook: ""
shares: Array(0)
tags: Array(0)
text: "<p>bar</p>"
title: "foo"
trash: false
```
![Screenshot_20200920_002101](https://user-images.githubusercontent.com/7738/93706023-3312bc80-fad7-11ea-8455-3800aa663258.png)

**Comment by dannyconnell** [comment]
Thanks @dantrevino . It seems that the `update` method isn't working when passing an array as a field value. When I comment out all of the fields that have arrays (attachments, shares & tags) it works. I'll try and get this fixed. Thanks!

**Comment by dannyconnell** [comment]
Fixed in 0.7.3 @dantrevino
